pkg/alertmanager/state_replication_test.go

```diff
@@ -163,6 +185,7 @@ func TestStateReplication_Settle(t *testing.T) {
 		name              string
 		replicationFactor int
 		read              readStateResult
+		states            map[string]alertspb.FullStateDesc
```

Comment:
[nit] I think renaming it to `storeStates` could help clarifying what it's used for.